core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -349,10 +349,10 @@ oppia.controller('CreatorDashboard', [
           canReviewActiveThread: function() {
             return $scope.canReviewActiveThread;
           },
-          stateName: function(){
+          stateName: function() {
```

Comment:
@apb7 This should be caught by the linter. Do you know why it was missed?

If it's a simple fix can we please update eslint to catch this?